hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/SharedNetworkLauncherSessionListener.java

```diff
@@ -140,22 +150,13 @@ public void testPlanExecutionStarted(@NonNull final TestPlan testPlan) {
                         case REPEATABLE -> EmbeddedNetwork.newSharedNetwork(EmbeddedMode.REPEATABLE);
                     };
             if (network != null) {
-                if (enableClprOverrides && network instanceof SubProcessNetwork subProcessNetwork) {
-                    subProcessNetwork.nodes().forEach(node -> {
-                        final var overrides = subProcessNetwork.getApplicationPropertyOverrides();
-                        final var existing = overrides.getOrDefault(node.getNodeId(), List.of());
-                        final var updated = new ArrayList<>(existing);
-                        for (int i = 0; i + 1 < updated.size(); i += 2) {
-                            if ("clpr.clprEnabled".equals(updated.get(i))) {
-                                updated.set(i + 1, "true");
-                                overrides.put(node.getNodeId(), List.copyOf(updated));
-                                return;
-                            }
-                        }
-                        updated.add("clpr.clprEnabled");
-                        updated.add("true");
-                        overrides.put(node.getNodeId(), List.copyOf(updated));
-                    });
+                if (network instanceof SubProcessNetwork subProcessNetwork) {
+                    if (enableClprOverrides) {
+                        upsertApplicationOverride(subProcessNetwork, "clpr.clprEnabled", "true");
+                    }
+                    if (hasRestartTests) {
+                        upsertApplicationOverride(subProcessNetwork, "quiescence.enabled", "true");
```

Comment:
Wondering why we can't just enable quiescence for the tests that actually use it 🤔 instead of baking it into the network launcher